client/src/components/RemainingTime.jsx

```diff
@@ -0,0 +1,31 @@
+import { PropTypes } from "prop-types";
+import React from "react";
+
+import Plural from "./Plural";
+
+const RemainingTime = ({ finalDate, currentDate = new Date() }) => {
+  const diffDates = finalDate.getTime() - currentDate.getTime();
+
+  if (diffDates <= 1000 * 60 * 60 * 24) {
+    const resInHours = diffDates / (1000 * 60 * 60);
+    return (
+      <span className={resInHours <= 3 ? "time-important" : ""}>
+        {resInHours >= 0 ? `${Math.round(resInHours)} h` : "bientôt"}
```

Comment:
I feel like "bientôt" is too generic. Can we show the number of minutes, or "Moins d'une heure" instead?

Reply:
"Moins d'une heure" will be good :)
That's not an exact science, the job runs every 3 hours